src/std/ucs2.c

```diff
@@ -80,7 +80,8 @@ double utod( const uchar *str, uchar **end ) {
 	}
 	buf[i] = 0;
 	result = strtod(buf,&bend);
-	*end = str + (bend - buf);
+	uchar *strcpy = ustrdup(str);
```

Comment:
Best avoid calling this after an existing C function?

Reply:
Yes, true, completely forgot about it.